gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageStubProvider.java

```diff
@@ -159,12 +159,12 @@ public boolean equals(Object o) {
         return false;
       }
       ChannelAndRequestCounter that = (ChannelAndRequestCounter) o;
-      return com.google.common.base.Objects.equal(channel, that.channel);
+      return Objects.equals(channel, that.channel);
```

Comment:
Why do we need above checks? Aren't they are covered by `Objects#equals` already?

Reply:
`Objects.equals` checks for channel object, but not `ChannelAndRequestCounter`,  we would want to delete the counter if the channel is same, but not necessarily the count, we would have access to channel alone to evict.